hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/NomisService.kt

```diff
@@ -33,22 +33,22 @@ class NomisService(
     getOffendersIds(page, pageSize).content
 
   fun getNomsNumberForBooking(bookingId: Long): String? = prisonApiWebClient.get()
-    .uri("/api/bookings/$bookingId?basicInfo=true")
+    .uri("/api/bookings/{bookingId}?basicInfo=true", bookingId)
```

Comment:
stop using kotlin interpolation and parameterize calls so that micrometer doesn't generate a separate tag for each uri (and therefore doesn't run out of memory)